### PR TITLE
add list structure

### PR DESCRIPTION
**Pull request by Pamphilemkp**
To do list app 
- list structure this is just the structuration of the to do list app 
built with html, css, Javascript 

additional tools
webpack, linters.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  {
+    description: 'Enjoy every single achievement',
+    completed: false,
+    index: 3,
+  },
+];
+
+// let itemFrominput = document.querySelector('.write-task ').value;
+
+// lists.unshift(itemFrominput)
```
Comment (vechicin) [inline src/index.js]: Please remove this commented code to keep a clean codebase.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+class Todo {
+  static addlist() {
+    const task = document.querySelector('.tasks');
+    const listitem = document.createElement('li');
+    // eslint-disable-next-line no-restricted-syntax
```
Comment (vechicin) [inline src/index.js]: Please do not disable ESlint.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+Todo.addlist();
+
+// line through the item when clicked
+const label = document.querySelectorAll('.label');
+
+label.forEach((label) => label.addEventListener('click', () => {
+  label.style.textDecoration = 'line-through';
+}));
```
Comment (vechicin) [inline src/index.js:45]: Screen Shot 2022-07-13 at 9 25 03

This part of your code doesn't work, please fix it.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Pamphilemkp ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Nice wireframe!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Pamphilemkp, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

 #### I will leave the comments here, since your `Readme file` is not accessible for this `PR` as you did not make any change on it ####

- [OPTIONAL] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. It is a good thing you are using the template from Microverse but please kindly adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your project.  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**